components/SearchSandbox/components/PreviewList.js

```diff
@@ -14,20 +16,23 @@ class PreviewList extends React.Component {
 				url,
 				title,
 				description,
+				listLayout,
 				image, // prettier-ignore
 			} = props.componentProps.metaFields;
 			this.state = {
 				url,
 				title,
 				description,
 				image,
+				listLayout,
 			};
 		} else {
 			this.state = {
 				title: '',
 				description: '',
 				image: '',
 				url: '',
+				listLayout: true,
```

Comment:
Instead of having a boolean prop - we should have a prop called `layout`, which can accept - `card` and `list` as its values.

Reply:
Sounds good, we can add more layout later if we want